src/model/cloud-runner/providers/k8s/kubernetes-task-runner.ts

```
@@ -22,94 +22,662 @@ class KubernetesTaskRunner {
         output += outputChunk;
 
         // split output chunk and handle per line
         for (const chunk of outputChunk.split(`\n`)) {
-          ({ shouldReadLogs, shouldCleanup, output } = FollowLogStreamService.handleIteration(
-            chunk,
-            shouldReadLogs,
-            shouldCleanup,
-            output,
-          ));
+          // Skip empty chunks and kubectl error messages (case-insensitive)
+          const lowerChunk = chunk.toLowerCase();
```

ESLint

[error] 'lowerChunk' is already declared in the upper scope on line 38 column 15.

(no-shadow)